### PR TITLE
Adds dragged file path to emited event

### PR DESCRIPTION
**Pull request by alanzanattadev**
Hi,

This PR adds file path to DragEvent of FileTreeItems, to let plugins developers create features based on file dragged from filetree and dropped in others panes. Atom file tree already set initialPath so I've only added the same line to keep a similar API to bind with.

Thank you

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @tyangliu, @shushz and @ssorallen to be potential reviewers.

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: pkg/nuclide-file-tree/components/FileTreeEntryComponent.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ export class FileTreeEntryComponent extends React.Component {
     const nativeEvent = (event.nativeEvent: any);
     nativeEvent.dataTransfer.effectAllowed = 'move';
     nativeEvent.dataTransfer.setDragImage(fileIcon, -8, -4);
-
+    nativeEvent.dataTransfer.setData("initialPath", this.props.node.uri);
```
Comment (tyangliu) [inline pkg/nuclide-file-tree/components/FileTreeEntryComponent.js]: looks good, minor nit - single quotes around the string to be consistent :)

Comment (alanzanattadev) [inline pkg/nuclide-file-tree/components/FileTreeEntryComponent.js]: done :)

**Comment by tyangliu** [comment]
Thanks for the change! Sorry for the extra trouble, but do you mind squashing the commits into just one?

**Comment by nmote** [comment]
@tyangliu @alanzanattadev Don't worry about squashing, our tooling should squash automatically.

**Comment by alanzanattadev** [comment]
Ok, thank you :)

**Comment by tyangliu** [comment]
Looks good then, thanks for the PR!

File: pkg/nuclide-file-tree/components/FileTreeEntryComponent.js (hunk 1 above)
Comment (nmote) [inline pkg/nuclide-file-tree/components/FileTreeEntryComponent.js]: For future contributions, you can use the `linter-eslint` Atom package to warn you of issues from within Atom. You can also run `npm run lint-all --ignore-scripts false` from the root of the `nuclide` repository to check for lint errors (this should probably be in CONTRIBUTING.md or something).

Our CI prevents changes with lint errors from getting committed so things need to be lint-clean to get accepted.

**Comment by nmote** [comment]
@facebook-github-bot shipit

**Comment by nmote** [comment]
@facebook-github-bot import

**Comment by nmote** [comment]
Oh, you need to sign the CLA

**Comment by alanzanattadev** [comment]
@nmote I've already signed it :/ I've tried again and it tells me I'm already registered, so what can I do ?

**Comment by nmote** [comment]
I'll look into it

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1759097961039591/int_phab) to review.

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1759097961039591/int_phab) to review.

**Comment by facebook-github-bot-9** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Pull request closed without merging**